tests/releaseNotes_Test.py

```diff
@@ -27,5 +28,28 @@ def test_givenSimpleInputPrintVersionBlockGetsCalled(self):
         releaseNotes.generateReleaseNotesByPromotedVersions()
         writer.printVersionBlock.assert_called_once_with({}, "1.0.1.2", "N/A", ["TCKT-1"])
 
+    def test_givenContentTemplate_ProducesCorrectResponse(self):
+
+        conf = {"ContentTemplateFile" : "dummy.txt" }
+        promotedVersions = {}
+
+        ticketProvider = Mock()
+        ticketProvider.extractTicketsFromMessage = MagicMock(return_value=["TCKT-1"])
+        ticketProvider.getTicketInfo = MagicMock(return_value={ "issue_type_icon" : "None", "html_url" : None,
+                                                                "ticket" : "TCKT-1",   "title" : "TCKT-1 Summary",
+                                                                "embedded_link" : {} })
+
+        writer = HtmlWriter(ticketProvider, "abc ${content} xyz")
```

Comment:
Current version doesn't support multiple writers so having the template as a global config was reasonable solution. I will make it writer-specific.